ssts/segmentation/segmentation_watershed.py

```diff
@@ -97,7 +97,7 @@ def fit_transform(self, data, outliers=None, pers_thresh=DEF_THRESH):
         return self.transform(data, outliers, pers_thresh)
 
     @staticmethod
-    def normalize_data(data):  # Maps to (0,1). This allows negative values to be grains.
+    def normalize_data(data):  # Maps from (-a,b) to (0,1). This allows negative values to be grains.
```

Comment:
The height property has range from (-a, b). This normalization step maps the height to the interval (0,1). Also the background gets converted to a peak for the watershed algorithm to segment it.